notesserver/highlight.py

```diff
@@ -39,13 +39,28 @@ def build_search_kwargs(self, *args, **kwargs):
             highlight_options = {
                 'fields': {
                     content_field: {'store': 'yes'},
-                }
+                    'tags': {'store': 'yes'},
+                },
             }
             if isinstance(highlight, dict):
                 highlight_options.update(highlight)
             res['highlight'] = highlight_options
         return res
 
+    def _process_results(
```

Comment:
I'll try that. I wasn't sure about the performance impact of doing highlighting outside of ES, but @dan-f is working on measuring that.